gobblin-metastore/src/test/java/org/apache/gobblin/metastore/testing/TestMetastoreDatabaseServer.java

```diff
@@ -83,10 +83,10 @@ class TestMetastoreDatabaseServer implements Closeable {
     if (this.embeddedMysqlEnabled) {
       mySQLContainer = new MySQLContainer<>("mysql:" + MYSQL_VERSION)
           .withUsername(this.dbUserName)
-          .withPassword(this.dbUserPassword)
-          .withEnv(MYSQL_ROOT_PASSWORD, "");
-
+          .withPassword(this.dbUserPassword);
       mySQLContainer.start();
+      mySQLContainer.withEnv(MYSQL_ROOT_PASSWORD_KEY, ROOT_PASSWORD);
```

Comment:
You may need to be defining this before running `.start()`

Reply:
No, the tests are running in local , only failing in CI pipeline because the root password in github actions is different than the user password. Actually, I guess, mysqlcontainer is not allowing the root password to be different than the user password. Have changed the root password in github action to be same as user. Have updated the PR

    if (this.password != null && !this.password.isEmpty()) {
      this.addEnv("MYSQL_PASSWORD", this.password);
      this.addEnv("MYSQL_ROOT_PASSWORD", this.password);